.github/workflows/android_arm64.yml

```diff
@@ -150,42 +145,67 @@ jobs:
       # Build
       #
 
-      - name: Build vcpkg
+      - name: Clone OneTBB
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        timeout-minutes: 15
+        with:
+          repository: 'oneapi-src/oneTBB'
+          path: ${{ env.TBB_SRC }}
+          submodules: 'true'
```

Comment:
So we want to test in pre-commit with the latest oneTBB from master? Sounds a bit risky (unfixed TBB version can break our pre-commits any time then)

I'd consider either fixing the commit via "ref" field here and updating it on demand; or moving the pipeline to nightly
But anyway if we absolutely have to proceed with the latest available oneTBB explicit setting "ref" to "master" is required to make sure the pipeline is working on release OV branch